sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.0
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.17.1
```

Comment:
Since we're already mucking about here, we should update all of these dependencies. Easiest way is to `go get -u all` followed by `go mod tidy`.

Reply:
Got it will execute.